kornia/augmentation/augmentation.py

```diff
@@ -1072,28 +1072,47 @@ def inverse(
             padding_size = [0, 0, 0, 0]
         return self.precrop_padding(out, padding_size)
 
+    def forward_parameters_precrop(self, batch_shape) -> Dict[str, torch.Tensor]:
```

Comment:
Actually this breaks the cases where you compute the parameters after padding because the input size has already changed and thus you would need ro recognize that in here and not recompute the padding etc on top. This is the reason I introduced the `precrop` thingy :')

Reply:
I see. Probably we should add a ```preprocessing``` hook before forwarding parameters in the ```AugmentationBase```? I want to avoid updating the ```AugmentationSequential``` since it is already a bit nasty.